### ISSUE TITLE
Recent updates broke mhchem integration?

### ISSUE THREAD

**Issue by vhsven** [yzhang-gh/vscode-markdown#1116]
This renders fine in v3.4.0 but not in v3.4.2: $\pu{kg m-1}$.

Environment: VSCode 1.66.2 on Windows 11

See also #521.

**Comment by yzhang-gh** [comment]
Thanks for catching this.

**Comment by yzhang-gh** [comment]
(Just a tip of installing previous versions before we fix this issue: https://github.com/microsoft/vscode/issues/12764#issuecomment-442370545)

**Comment by Lemmingh** [comment]
Caused by module resolution. Our CI has been warning about it for months.

Hint:
KaTeX is an explicit dual-module package, aka "hybrid package". Make sure to always resolve to the same module system. Should be super easy to fix.

**Comment by yzhang-gh** [comment]
Do not have enough time to dive in, but `require('katex/dist/contrib/mhchem')` seems to work. Not sure whether it is because of webpack 5.

**Comment by Lemmingh** [comment]
Since version 0.14, KaTeX has declared a [conditional exports map](https://nodejs.org/api/packages.html#conditional-exports):

<https://github.com/KaTeX/KaTeX/blob/b1311aff1bfeebd75df9389aadfba7a4a697b203/package.json#L15-L18>

Then, the **canonical** way to import the CJS format of `mhchem` is:

```javascript
require("katex/contrib/mhchem")
```

Any other statement violates the spec, and may confuse webpack.

Extra note:
TypeScript still cannot properly understand the `exports` field yet.

---

The [`katex/contrib/mhchem/mhchem.js`](https://github.com/KaTeX/KaTeX/blob/b1311aff1bfeebd75df9389aadfba7a4a697b203/contrib/mhchem/mhchem.js) is an ES module. However,

```javascript
require("katex/contrib/mhchem/mhchem")
```

used to work because KaTeX wasn't explicit, allowing webpack to treat `import` as `require`.

**Comment by yzhang-gh** [comment]
has been fixed in v3.4.3
